packages/mds-db/geographies.ts

```diff
@@ -65,6 +61,46 @@ export async function readGeographies(params?: {
     throw err
   }
 }
+
+export async function readGeographiesSummary(params?: {
+  get_read_only?: boolean
+}): Promise<GeographySummary[]> {
+  // use params to filter
+  // query on ids
+  // return geographies
+  try {
+    const client = await getReadOnlyClient()
+
+    const cols =[...schema.TABLE_COLUMNS.geographies].filter(col => col !== schema.COLUMN.geography_json).join(',')
+
+    let sql = `select ${cols} from ${schema.TABLE.geographies}`
+
+    const conditions = []
+    const vals = new SqlVals()
+
+    if (params && params.get_read_only) {
+      conditions.push(`read_only IS TRUE`)
+    }
+
+    if (conditions.length) {
+      sql += ` WHERE ${conditions.join(' AND ')}`
+    }
+
+    const values = vals.values()
+    // TODO insufficiently general
+    // TODO add 'count'
+    const { rows } = await client.query(sql, values)
```

Comment:
I am not sure if there would be performance concerns, but this method _could_ just delegate to `await readGeographies` and then map out the `geography_json`.

Reply:
I agree with @drtyh2o , this is a lot of needlessly duplicated code. I don't think there's going to be a lot of geographies so I wouldn't be too concerned about performance right now.